draft-ietf-quic-transport.md

```diff
@@ -1232,6 +1232,15 @@ client MUST terminate with a VERSION_NEGOTIATION_ERROR error code if
 version negotiation occurred but it would have selected a different version
 based on the value of the supported_versions list.
 
+When an endpoint accepts multiple QUIC versions, it can potentially interpret
+transport parameters as they are defined by any of the QUIC versions it
+supports. Since the QUIC protocol relies on transport parameters for
+authenticating the version, the position and the format of the version fields in
+transport parameters MUST be consistent across all QUIC versions supported by
+the client. If a version of QUIC needs to define a new format for transport
+parameters, it MUST define and use a new TLS extension for transport parameters
```

Comment:
One request: The rest of the section on transport parameters doesn't mention TLS.  Can we change this to "a new handshake extension" or something similar to avoid adding new references to it.